.github/workflows/build.yml

```diff
@@ -16,3 +16,9 @@ jobs:
         uses: GabrielBB/xvfb-action@v1.4
         with:
           run: npm test
+      - name: Validate codecov.yml
+        if: matrix.os == 'ubuntu-latest'
+        run: curl --data-binary @.github/codecov.yml https://codecov.io/validate
+      - name: Codecov
+        if: matrix.os == 'ubuntu-latest'
+        run: bash <(curl -s https://codecov.io/bash) -f coverage/coverage.json || echo 'Codecov upload failed'
```

Comment:
CodeCov has a github action that may help https://github.com/coverallsapp/github-action

Reply:
The issue now is not executing Codecov, but actually gathering code coverage. Once that works, we can probably replace the Codecov submission with a GitHub Action.